src/state/quick-open-mixin.ts

```diff
@@ -0,0 +1,38 @@
+import type { QuickOpenDialogParams } from "../dialogs/quick-open/ha-quick-open-dialog";
+import type { Constructor, PropertyValues } from "lit-element";
+import { HassElement } from "./hass-element";
+import { showQuickOpenDialog } from "../dialogs/quick-open/show-dialog-quick-open";
+
+declare global {
+  interface HASSDomEvents {
+    "hass-quick-open": QuickOpenDialogParams;
+    "hass-service-called": HassServiceCalledParams;
+  }
+}
+
+export interface HassServiceCalledParams {
+  domain: string;
+  service: string;
+  serviceData?: {};
+}
```

Comment:
This is unused?

Reply:
Not unused, but maybe improperly placed?

I'm calling `fireEvent(this, "hass-service-called", eventData);` after user clicks one of the Reload actions, but the event type wasn't a valid `HassEvent`. We currently only fire that event from [call-service-button.js](https://github.com/home-assistant/frontend/blob/c2fba15fc6d1308ddc583514a4926bd451e07ade/src/components/buttons/ha-call-service-button.js#L79) which doesn't care about typings.

I'm definitely open to suggestions on a better approach here.

Reply:
Why do we fire that event? Don't think that is needed/used?

Reply:
There are several areas of the UI that are listening for this event:

<img src="https://user-images.githubusercontent.com/1480827/95265414-01436a80-07e6-11eb-97f1-e801e5d701aa.png" width="75%" />
 
Figured that since we can open this dialog from anywhere in the UI, it would ensure consistent panel reactions.

Reply:
Looked into it and yeah... this event fire is unnecessary. Removed it